go-controller/pkg/ovn/egressip_test.go

```diff
@@ -275,12 +334,6 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 						Items: []v1.Namespace{*egressNamespace},
 					})
 
-				fakeOvn.fakeExec.AddFakeCmdsNoOutputNoError(
-					[]string{
-						fmt.Sprintf("ovn-nbctl --timeout=15 set logical_switch_port etor-GR_node1 options:nat-addresses=router"),
-					},
-				)
-
```

Comment:
You are removing this check, you should be replacing it. I.e: add a check before any node update that the option is set on `node1`. Later, at the end of the test the option should be removed from `node1` and added on `node2`

Reply:
done